bugbot/rules/file_crash_bug.py

```diff
@@ -86,6 +86,18 @@ def get_bugs(self, date):
             "Firefox", "nightly"
         )
         signatures = data_fetcher.analyze()
+        # This is the last filtering stage which aims to avoid filling bugs for
+        # junky crashes, where the volume is low and the crashes do not show
+        # signals of being actionable or critical.
+        signatures = [
+            signature
+            for signature in signatures
+            if signature.num_installs > 20
+            or signature.num_crashes > 200
+            or signature.is_potential_near_null_crash
+            or signature.is_potential_security_crash
+            or signature.has_moz_crash_reason("MOZ_RELEASE_ASSERT")
```

Comment:
Hmm possibly some other crash reasons would be interesting here, but we can try this. It at least avoids all of the OOM and shutdown hangs we don't really care about.

Reply:
OOM and shutdown hangs crashes are already filtered out in the first step:
https://github.com/mozilla/bugbot/blob/160c0017ed7c054834296e2fe4ed6dbada0a9fef/bugbot/crash/analyzer.py#L598-L623

Reply:
>possibly some other crash reasons would be interesting here

If you have specific crash reasons in mind, we can add them here so we still file bugs for them even when they have a small volume.

Reply:
Not particularly. I see some MOZ_CRASH() but they are mostly not interesting. There's one about DocumentChannel::SetLoadFlags that is potentially interesting. But I think this is okay for now.

Reply:
Thank you! I added it in 1e01d1f89215b4d8e9f2190d01ff14eac5b642ae. We can always tweak the list later based on the feedback.